spec/lib/insights/api/common/filter_spec.rb

```diff
@@ -0,0 +1,74 @@
+describe Insights::API::Common::Filter do
+  let(:this) { described_class }
+
+  context "compact_filter method" do
+    it "compacts association filter specifications of string" do
+      parameters  = {"association" => {"attribute" => "value"}}
+      expectation = {"association.attribute" => "value"}
+
+      expect(this.public_send(:compact_filter, parameters)).to(eq(expectation))
```

Comment:
Is there an advantage to doing `this.public_send(:compact_filter, parameters)` rather than `described_class.compact_filter(parameters)`?

Reply:
Nope, you're right. Updating to just described_class.compact_filter(...)